integration-tests/tests/test_eth_api.py

```diff
@@ -0,0 +1,55 @@
+import random
+import pytest
+import jsonschema
+import json
+from conftest import option
+from test_cases import EthApiTestCase
+
+
+@pytest.mark.rpc
+@pytest.mark.ethclient
+class TestRpc(EthApiTestCase):
+    @pytest.mark.parametrize(
+        "iterations", range(3) # Run multiple times, each with a different Tx
+    )
+
+    def test_(self, iterations):
+        _id = str(random.randint(1, 8888))
+
+        __ = self.rpc_valid_request("ethclient_blockNumber", [self.network_id], _id)
+
+        __ = self.rpc_valid_request("ethclient_suggestGasPrice", [self.network_id], _id)
```

Comment:
Why do you create a variable here if it's unused?
```suggestion
        self.rpc_valid_request("ethclient_blockNumber", [self.network_id], _id)

        self.rpc_valid_request("ethclient_suggestGasPrice", [self.network_id], _id)
```

Reply:
Sorry, I'm used from other languages to explicitly ignore returned values. Will remove it.